backend/tests/integration/orcid/test_orcid_domain_models.py.orig

```diff
@@ -0,0 +1,561 @@
+# -*- coding: utf-8 -*-
```

Comment:
why is this part of this commit?

Reply:
Sorry, I made a mess with the commit, I will fix it.